source/renderer/app/components/sidebar/wallets/WalletSearch.scss

```diff
@@ -0,0 +1,71 @@
+.component {
+  align-items: center;
+
+  border-bottom: 1px solid rgba(250, 251, 252, 0.1);
+  display: flex;
+  position: relative;
+
+  &:focus,
+  &:active,
+  &:focus-within {
+    border-bottom-color: rgba(250, 251, 252, 0.5);
+
+    .searchIcon {
+      path {
+        fill: var(--theme-sidebar-menu-add-button-text-color);
+      }
```

Comment:
Ideally we could modify icon colors with just `color` css property e.g. if we change `fill="currentColor"` in svgs.